department-of-veterans-affairs/va.gov-cms
# VAGOV-1158 content export tar

## Description

See #1158.  A new endpoint is added at `/export/content.tar` which provides a 302 to the actual file.

This also includes the code from https://github.com/department-of-veterans-affairs/va.gov-cms/pull/827.

## Testing done

* Create a tar of the `export-content` directory.
* Set the config variable `lando drush config:set va_gov_content_export.settings content_tar_uri public://export-content/content.tar`
* Visit `/export/content.tar`.  See the 302 to `/sites/default/files/content.tar`
* Remvoe the `content.tar` file.
* Visist `export/content.tar` and get a 403.

## Definition of Done


---
This change is [<img src="https://reviewable.io/review_button.svg" height="34" align="absmiddle" alt="Reviewable"/>](https://reviewable.io/reviews/department-of-veterans-affairs/va.gov-cms/1197)
